CreateInstaller.ps1

```diff
@@ -0,0 +1,12 @@
+$tempDirectoryName = '.\__squirrel_temp__'
+
+if ($DTE -eq $null) {
+  echo "Run this from the NuGet Package Console inside VS"
+  exit 1
+}
+
+mkdir -Path $tempDirectoryName
+rm -r -fo "$tempDirectoryName\*.nupkg"
+NuGet pack .\src\AudioSwitcher\AudioSwitcher.csproj -OutputDirectory "$tempDirectoryName" -Prop Configuration=Release
```

Comment:
Is the NuGet.CommandLine supposed to put NuGet.exe on the path in the Package Manager Console? I ran the script from the Package Manager Console and it failed to find it.